index.qmd

```diff
@@ -140,8 +203,8 @@ variables @steyerberg2001
 
 
 #### Random Forest
+The random forest plot is an extension of the decision tree methodology to reduce the variance. Decision trees are very sensitive to the training data set and can lead to high variance; thus potential issues with generalisation of the model. The random forest plot selects random observation of the data set to create multiple decision trees. Random variables are selected for each tree to use in the training of the data set. The aggregated output of the generated decision trees is then used to create an estimate. The random forest methology is associated with reduced variance.
```

Comment:
Random forest isn't a plot, although you can plot individual forests that are generated by random forests.

The point of Random Forests is that they avoid over-fitting which can arise when you use a single decision tree, since as you've clocked random samples are selected from the training data set and a decision tree fitted to each.

It would be worth explaining to readers how decision trees work with classification in that they select one of the predictor variables that optimally splits the cohort into the outcome of interest (which in this case is whether a patient has malignant or benign tumors). This is then repeated in each of the subsets using another of the predictors and the process is repeated until the user chosen "depth" of the tree is found.

Random Forests take this ensemble of decision trees made on subsets of the training data and then aggregate the results.